python/dask_cudf/dask_cudf/backends.py

```diff
@@ -33,3 +33,33 @@ def concat_cudf(
     assert axis == 0
     assert join == "outer"
     return cudf.concat(dfs)
+
+
+try:
+
+    from dask.dataframe.methods import group_split, hash_df
+    import cudf._lib as libcudf
+
+    @hash_df.register(cudf.DataFrame)
+    def hash_df_cudf(dfs):
+        return dfs.hash_columns()
+
+    @hash_df.register(cudf.Index)
+    def hash_df_cudf_index(ind):
+        from cudf.core.column import column, numerical
+
+        cols = [column.as_column(ind)]
+        return cudf.Series(numerical.column_hash_values(*cols))
+
+    @group_split.register((cudf.DataFrame, cudf.Series, cudf.Index))
+    def group_split_cudf(df, c, k):
+        source = [df[col] for col in df.columns]
+        # TODO: Use proper python API (#2807)
```

Comment:
Do we want to wait on #2807 to be merged?

Reply:
No - I can submit a mini PR to update the code here when #2807 goes through